projects/core/src/auth/http-interceptors/auth-error.interceptor.ts

```diff
@@ -30,7 +30,7 @@ export class AuthErrorInterceptor implements HttpInterceptor {
     csagentErrorHandlingService: CustomerSupportAgentErrorHandlingService
   );
   /**
-   * @deprecated since version 1.x
+   * @deprecated since version 1.3
    * Instead, use constructor(
    * userErrorHandlingService: UserErrorHandlingService,
    * clientErrorHandlingService: ClientErrorHandlingService,
```

Comment:
sometimes, we also add ticket number. 
```
/**
   * @deprecated since 1.x
   * NOTE: check issue:#xxx for more info
   *
   * TODO(issue:#xxx) Deprecated since 1.x
   */
```